lms/djangoapps/course_api/blocks/api.py

```diff
@@ -59,7 +59,12 @@ def get_blocks(
 
     if user is not None:
         transformers += COURSE_BLOCK_ACCESS_TRANSFORMERS
-        transformers += [MilestonesAndSpecialExamsTransformer(include_special_exams), HiddenContentTransformer()]
+        transformers += [
+            MilestonesAndSpecialExamsTransformer(include_special_exams),
+            HiddenContentTransformer(),
+            OverrideDataTransformer(user)
```

Comment:
Can the `OverrideDataTransformer` be added to this list conditionally, in the same way that per-student overrides are controlled by settings in the courseware? I don't believe it's gone through load testing, so we'll want to keep it off by default (I believe that's the current behavior for courseware as well?).

Reply:
should i this add transformer with the `"INDIVIDUAL_DUE_DATES": true`?

Reply:
Right now, the courseware applies the `IndividualStudentOverrideProvider` in aws.py like this:

```python
##### Individual Due Date Extensions #####
if FEATURES.get('INDIVIDUAL_DUE_DATES'):
    FIELD_OVERRIDE_PROVIDERS += (
        'courseware.student_field_overrides.IndividualStudentOverrideProvider',
    )
```

I think that probably the right thing to do is to check `settings.FIELD_OVERRIDE_PROVIDERS` to see if it has the `IndividualStudentOverrideProvider` in it, but I'm fine with checking for `INDIVIDUAL_DUE_DATES` as well. There shouldn't be a need for a new flag. And my understanding is that CCX overrides already happen via a different mechanism (since they are based on the `course_id` and not the individual student), so it would be inappropriate to check for `CUSTOM_COURSES_EDX`.

Reply:
@ormsbee added check for `IndividualStudentOverrideProvider`